pip_review/__main__.py

```diff
@@ -86,6 +86,9 @@ def parse_args():
     parser.add_argument(
         '--freeze-outdated-packages', action='store_true', default=False,
         help='Freeze all outdated packages to "requirements.txt" before upgrading them')
+    parser.add_argument(
+        '--tabulated', '-t', action='store_true', default=False,
+        help='Print the outdated packages in tabbular format')
```

Comment:
"Tabulated" and "tabular format" (note the single 'b') are roundabout ways to say "table". Easy does it, so all else being equal, I would just go "table":

```suggestion
        '--table', '-t', action='store_true', default=False,
        help='Print the outdated packages in table format')
```

*However*, we cannot use the `-t` short option (at least not without it being a breaking change), because `pip install` has a `-t` for `--target` which some users might be currently forwarding through pip-review. The other pull request used `-p`, which does not have this problem. That `p` stood for `--preview`, but you can also stick with the long option `--table`, which is less consistent with the short option, but does describe the feature better in my opinion.